docs/_snippets/storybook-config-layout.md

```diff
@@ -11,6 +11,14 @@ addons.setConfig({
   theme: undefined,
   selectedPanel: undefined,
   initialActive: 'sidebar',
+  layoutCustomisations: {
```

Comment:
@Sidnioulz, if you're ok, can you adjust the table to which this example relates in the [features and behavior documentation](https://github.com/storybookjs/storybook/blob/next/docs/configure/user-interface/features-and-behavior.mdx) to include the `layoutCustomisations` option?

Reply:
@jonniebigodes I'm not sure I'm understanding you. Are you asking me to add `layoutCustomisations` to the prop tables in `features-and-behavior.mdx`? To my understanding, I have added them.